src/blocks-11.js

```diff
@@ -3,11 +3,15 @@ import { registerBlock } from './utils/helper';
 // Register Blocks
 import * as postCarousel from './blocks/post-carousel';
 import * as posts from './blocks/posts';
+import * as testimonial from './blocks/testimonials/testimonial';
+import * as testimonials from './blocks/testimonials';
 
 /**
  * Function to register blocks provided by CoBlocks.
  */
 [
 	posts,
 	postCarousel,
+	testimonial,
+	testimonials,
```

Comment:
Did you happen to check the chunk size for this yet? I wonder if we might just want to split into another blocks chunk. I ended up doing that for the Counter block due to chunk size.

Reply:
I'll double check that at the end once my code is good enough to merge :)